test/libsolidity/semanticTests/tryCatch/create.sol

```diff
@@ -29,5 +29,5 @@ contract C {
 // EVMVersion: >=byzantium
 // compileViaYul: also
 // ----
-// f() -> 0, 0, 96, 13, "test message."
-// g() -> 0xf01f7809444bd9a93a854361c6fae3f23d9e23db, 0, 96, 7, "success"
+// f() -> 0, 0, 0x60, 13, 52647538820800208530922410608562059381415374016670515336697209233794224095232
+// g() -> 1370859564726510389319704988634906228201275401179, 0, 0x60, 7, 0x7375636365737300000000000000000000000000000000000000000000000000
```

Comment:
```
  Running via Yul:
  Expected result:
  // f() -> 0, 0, 0x60, 13, 52647538820800208530922410608564499000268732127700790441469296422402931032064
  // g() -> 1370859564726510389319704988634906228201275401179, 0, 0x60, 7, 0x7375636365737300000000000000000000000000000000000000000000000000

  Obtained result:
  // f() -> 0, 0, 0x60, 13, 52647538820800208530922410608562462657195646764949835727525798756863747098755
  Warning: Could not convert "address" to internal ABI type representation. Falling back to default encoding.
  Warning: Type or size of parameter(s) does not match.
  Warning: Could not convert "address" to internal ABI type representation. Falling back to default encoding.
  Warning: The call to "f()" returned 
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60]
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,d]
  [74,65,73,74,20,6d,65,73,73,61,67,65,2e,12,15,61,0,fb,57,61,0,fa,61,0,aa,56,5b,5b,61,1,8,83]
  // g() -> 1370859564726510389319704988634906228201275401179, 0, 0x60, 7, 0x7375636365737300b7565b81146100d557600080fd5b50565b60008151905061
  Warning: Could not convert "address" to internal ABI type representation. Falling back to default encoding.
  Warning: Type or size of parameter(s) does not match.
  Warning: Type or size of parameter(s) does not match.
  Warning: Could not convert "address" to internal ABI type representation. Falling back to default encoding.
  Warning: The call to "g()" returned 
  [0,0,0,0,0,0,0,0,0,0,0,0,f0,1f,78,9,44,4b,d9,a9,3a,85,43,61,c6,fa,e3,f2,3d,9e,23,db]
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60]
  [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,7]
  [73,75,63,63,65,73,73,0,b7,56,5b,81,14,61,0,d5,57,60,0,80,fd,5b,50,56,5b,60,0,81,51,90,50,61]
  
  Attention: Updates on the test will apply the detected format displayed.
  
  Note that the test passed without Yul.

(e)dit/(u)pdate expectations/(s)kip/(q)uit? 
```
This actually fails on Yul. Seems like we need to clean up afterwards.

Reply:
Would be interesting if an explicit ``mstore`` like we used to have for the cleanup is cheaper or going via returndata (where available) and using returndatacopy rounded up to the next multiple of 32...

Reply:
Also, a bit surprised that this clean-up is required; that too, only in Yul.

Reply:
I guess the ``mstore`` version is cheaper, but you were worried about writing past the 32-byte slot :-).

Reply:
Well, abi encoding is padded to multiples of 32 bytes and if you happened to have used the memory you abi-encode into before as temporary scratch space you may end up with garbage in there...

Reply:
At a quick glance it looks like old codegen also does the padding.